includes/functions.php

```diff
@@ -2397,7 +2397,11 @@ function cache_peeringdb()
                         }
                     }
                     $pdb_ix_peers_ids = implode(',', $peer_keep);
-                    dbDelete('pdb_ix_peers', "`pdb_ix_peers_id` NOT IN ($pdb_ix_peers_ids)");
+                    if (empty($pdb_ix_peers_ids)) {
+                        dbDelete('pdb_ix_peers');
```

Comment:
I think this should be restricted to `ix_id`

Reply:
I noticed that, the query was broken before.  Not sure they were ever deleting anything.

I can either restrict it to ix_id and do multiple deletes or collect all valid ids and do one delete per table.  I started coding the second option, but ran out of time.